lib/associations/belongs-to-many.js

```diff
@@ -449,6 +449,13 @@ BelongsToMany.prototype.injectGetter = function(obj) {
         _.assign(throughWhere, through.scope);
       }
 
+      //If a user pass a where on the options through options, make an "and" with the current throughWhere
+      if (options.through && options.through.where)
+        throughWhere = { $and : [
+            throughWhere,
+            options.through.where
+        ]};
+        
```

Comment:
Please add brackets around the if statement.
Please place `$and` on its own line in the object.

No spacing after `$and` before `:`.